LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+The MIT License
 
-Copyright (c) 2014 Chris Forrette
+Copyright (c) 2013-2016 Plaid Technologies, Inc. <support@plaid.com>
```

Comment:
2013? The [initial commit][1] was in 2014.


[1]: https://github.com/plaid/plaid-python/commit/eaefc152be977635ed7327b8d3a8494ae80ef8b2